components/display/ssd1325.rst

```diff
@@ -60,6 +65,68 @@ Configuration variables:
 - **pages** (*Optional*, list): Show pages instead of a single lambda. See :ref:`display-pages`.
 - **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
 
+Configuration examples
+**********************
+
+As of version 1.15, ESPHome supports grayscale (and color) displays. To utilize the grayscale capabilities
+of this display module, you'll likely want to add a ``color:`` section to your YAML configuration; please see
+:ref:`color <config-color>` for more detail on this configuration section. As this is a grayscale display, it
+only uses the white color element (as opposed to red, green, and/or blue).
```

Comment:
```suggestion
To utilize the grayscale capabilities of this display add a ``color:`` section to your YAML configuration; see
:ref:`color <config-color>` for more details. As this is a grayscale display, it only uses the white color element.
```

I don't see why we need to explain this is a new feature, docs should be about current version, unless we broke something may be, just may be

Reply:
you're welcome to update the changelog to mention this

Reply:
Thanks @glmnet! I made this change and added another code example, as well.